cmd/piper.go

```diff
@@ -369,6 +375,11 @@ func PrepareConfig(cmd *cobra.Command, metadata *config.StepData, stepName strin
 	}
 	myConfig.SetVaultCredentials(GeneralConfig.VaultRoleID, GeneralConfig.VaultRoleSecretID, GeneralConfig.VaultToken)
 
+	if GeneralConfig.TrustEngineToken == "" {
```

Comment:
The token can be set in the config?

Reply:
Nope, I've removed the if-condition now.